meta-geico/conf/machine/quartz.conf

```diff
@@ -0,0 +1,26 @@
+KMACHINE = "aspeed"
+KERNEL_DEVICETREE = "aspeed/quartz.dtb"
+UBOOT_MACHINE = "ast2600_openbmc_spl_defconfig"
+UBOOT_DEVICETREE = "ast2600-dcscm"
+SPL_BINARY = "spl/u-boot-spl.bin"
+
+# Disable hardware root of trust by default.
+SOCSEC_SIGN_ENABLE = "0"
+
+# Flash size is 64M
+FLASH_SIZE = "65536"
+
+require conf/machine/include/ast2600.inc
+require conf/machine/include/obmc-bsp-common.inc
+
+SERIAL_CONSOLES = "115200;ttyS4"
```

Comment:
Serial console should be an interpolated value

Reply:
This is yocto variables and format